### PR TITLE
fix: missing left margin on user dropdown in shell

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

Before: 
Screenshot 2023-06-19 at 11 15 11

After: 
Screenshot 2023-06-19 at 11 14 39

Introduced by: https://github.com/calcom/cal.com/pull/9581

## Type of change
- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/features/shell/Shell.tsx [hunk 1]
```diff
@@ -336,46 +336,44 @@ function UserDropdown({ small }: UserDropdownProps) {
   }
   return (
     <Dropdown open={menuOpen}>
-      <div className="ltr:md:-ml-5 rtl:md:-mr-5">
```
Comment (CarinaWolli) [inline packages/features/shell/Shell.tsx:339]: this div got removed

File: packages/features/shell/Shell.tsx [hunk 2]
```diff
@@ -336,46 +336,44 @@ function UserDropdown({ small }: UserDropdownProps) {
...
-          <button
+      <DropdownMenuTrigger asChild onClick={() => setMenuOpen((menuOpen) => !menuOpen)}>
+        <button
+          className={classNames(
+            "hover:bg-emphasis group mx-0 flex cursor-pointer appearance-none items-center rounded-full text-left outline-none focus:outline-none focus:ring-0 md:rounded-none lg:rounded",
+            small ? "p-2" : "px-2 py-1"
+          )}>
+          <span
             className={classNames(
-              "hover:bg-emphasis group mx-0 ml-0.5 flex cursor-pointer appearance-none items-center rounded-full text-left outline-none focus:outline-none focus:ring-0 md:rounded-none lg:rounded",
```
Comment (CarinaWolli) [inline packages/features/shell/Shell.tsx:343]: `ml-0.5` got removed

**Review by CarinaWolli — COMMENTED** [review]
changes are marked with comments, everything else stayed the same

**Review by sean-brydon — APPROVED** [review]
Tested - good job @CarinaWolli

**Pull request merged**